### DISCUSSION TITLE
lot of questions for a crossing

### DISCUSSION THREAD

**Discussion by ktonekaboni** [streetcomplete/StreetComplete#5897] [Feedback]
Hi, I just started using streetcomplete and it's a great fun. Only editing details of a crossing is sometimes annoying because I'm asked the same questions for each side of the crossing (four times) and five questions, so, 20 questions total..

Is it possible to have the option to answer them groupwise if it's the same for the complete crossing?

![Screenshot_20240912-183510](https://github.com/user-attachments/assets/83d43a70-61d0-4fcb-ba75-52396afe991a)

**Comment by mnalis** [comment]
> Is it possible to have the option to answer them groupwise if it's the same for the complete crossing?

No, see [quest guidelines](https://github.com/streetcomplete/StreetComplete/blob/master/QUEST_GUIDELINES.md): _"⚛️ Atomic quests: Per quest, only one thing should need to be answered by the user."_

Also those discussions:
- https://github.com/streetcomplete/StreetComplete/discussions/4761
- https://github.com/streetcomplete/StreetComplete/issues/2996

**Reply by matkoniecz** [reply]
and for why: it would be quite troublesome to get it working well without making it overly complex, confusing or encouraging to blindly mass answer :(

**Reply by mnalis** [reply]
As for the somewhat-related workaround, see `Settings` / `Quest selection and display order` - if you find that **solving some quests starts looking more like work then like fun, I recommend just disabling them** immediately without regret (as someone else will solve them later). In the longer run, such culling action will benefit both you and OSM (as you'll give up on using StreetComplete if become too tiresome)! 
 
You can even create one preset with them and one without (via `Manage presets`) so you can quickly switching between them depending on the mood (i.e. enabling "many quests" preset when you're having a lots of time on your hands and nothing better to do, and switching to "important quests only" for when other times when you're not feeling like solving them)

**Reply by ktonekaboni** [reply]
Thanks for the explanation. I know I can disable the categories. I love to answer them. I was just thinking how to make it more efficient.
